.github/workflows/test-suite-desktop-e2e-nightly.yml

```
@@ -0,0 +1,68 @@
+        with:
+          node-version-file: ".nvmrc"
+          cache: yarn
+
+      - name: Build desktop app
+        uses: ./.github/actions/build-desktop
+
+      - name: Pull Docker Images
+        shell: bash
+        env:
+          COMPOSE_FILE: ./docker/docker-compose.suite-ci-e2e.yml
+        run: docker compose pull ${{ inputs.containers }}
```

actionlint (1.7.4)

property "containers" is not defined in object type {}

(expression)